hw1/02-palindrome.js

```diff
@@ -1,3 +1,46 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", checkPalindrome);
+
+function checkPalindrome() {
+  const value = document.querySelector("input").value;
+
+  if (value != "") {
+    if (isNaN(value)) {
+      changeStatus("Input is not a number, please try again.", false);
+      return 1;
```

Comment:
What is the significance of the return value? I can't find any documentation on whether it means anything for an input event listener.

If it's not being used, I advise to remove the value and just have `return;`

Reply:
I think sometimes it is used to return a success for 0 or failure for 1, but it isn't necessary in this case so I'll leave it out with just a return for now